pkg/pipelines/pipelines_test.go

```diff
@@ -273,3 +271,59 @@ func TestPipelineActivityStatus(t *testing.T) {
 		require.Equal(t, v.expectedStatus, pa.Spec.Status.String())
 	}
 }
+
+var activityMessageTestCases = []struct {
+	description     string
+	folder          string
+	name            string
+	expectedMessage string
+}{
+	{
+		description:     "Jenkins X PipelineActivity has been successful run and message exists",
+		folder:          "message/success",
+		name:            "rajatgupta24-quickstart-004-pr-1-7",
+		expectedMessage: `Tasks Completed: 1 (Failed: 0, Cancelled 0), Skipped: 0`,
+	},
+	{
+		description:     "Jenkins X PipelineActivity has been timedout and message exists",
+		folder:          "message/timeout",
+		name:            "rajatgupta24-quickstart-004-pr-1-7",
+		expectedMessage: `PipelineActivity "rajatgupta24-quickstart-004-pr-1-7" failed to finish within "1h0m0s"`,
+	},
+	{
+		description:     "Jenkins X PipelineActivity has been cancelled and message exists",
+		folder:          "message/cancelled",
+		name:            "rajatgupta24-quickstart-004-pr-2-7",
+		expectedMessage: `PipelineActivity "rajatgupta24-quickstart-004-pr-2-7" was cancelled`,
+	},
+	{
+		description:     "Jenkins X PipelineActivity has two steps, one step that been timedout and other Succeeded",
+		folder:          "message/timedout-succeeded",
+		name:            "rajatgupta24-quickstart-004-pr-2-7",
```

Comment:
make the names more generic.

Reply:
I replaced it with `jx-test-project-repo`